### ISSUE TITLE
Add a JUnit reporter

### ISSUE THREAD

**Issue by grtjn** [MarkLogic-Attic/performance-meters#1]
It is very easy to add a JUnit reporter class that writes some simple JUnit compatible XML reports, which can be further processed with the built-in JUnit support in Ant to generate nice HTML reports with test results.

I have some old code which is still worth sharing..

**Comment by mblakele** [comment]
Sounds like a good idea. Can you create a pull request?

**Comment by grtjn** [comment]
Working on it, but strangely it looks like the performance-meters code on github is older than what I had worked against in the past. It seems to have come from SVN as well. Does that make sense?

**Comment by mblakele** [comment]
Yes, the code used to be in svn hosted by marklogic directly. It's possible that something went wrong when it was transferred to github.

If you'd like me to take a look, maybe you could create a github repository with your copy of the code, but preferably without the JUnit extras?

**Comment by grtjn** [comment]
I'll open a new ticket and second branch for it, but that will have to wait till next week..

Sent from Samsung Mobile
